src/adapters/defi.ts

```diff
@@ -130,7 +130,10 @@ class Defi {
     })
 
     const json = await resp.json()
-    if (json.error !== undefined) {
+    if (
```

Comment:
should create `InsufficientBalanceError` and throw that if error message includes "balance is not enough"
check `DiscordWalletTransferError.ts` for reference